packages/react-core/src/demos/examples/JumpLinks/JumpLinksWithDrawer.js

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import {
+  Button,
+  Drawer,
+  DrawerPanelContent,
+  DrawerContent,
+  DrawerContentBody,
+  DrawerHead,
+  DrawerActions,
+  DrawerCloseButton,
+  JumpLinks,
+  JumpLinksItem,
+  JumpLinksList,
+  PageSection,
+  Sidebar,
+  SidebarContent,
+  SidebarPanel,
+  TextContent,
+  getResizeObserver
+} from '@patternfly/react-core';
+import DashboardWrapper from '../DashboardWrapper';
+
+export const JumpLinksWithDrawer = () => {
+  const headings = ['First', 'Second', 'Third', 'Fourth', 'Fifth'];
+
+  const [offsetHeight, setOffsetHeight] = React.useState(0);
+  const [isExpanded, setIsExpanded] = React.useState(false);
+  const drawerRef = React.useRef();
+
+  React.useEffect(() => {
+    const masthead = document.getElementsByClassName('pf-c-masthead')[0];
+    const drawerToggleSection = document.getElementsByClassName('pf-c-page__main-section')[0];
+
+    getResizeObserver(masthead, () => {
+      setOffsetHeight(masthead.offsetHeight + drawerToggleSection.offsetHeight);
+    });
+  }, []);
+
+  const onCloseClick = () => {
+    setIsExpanded(false);
+  };
+
+  const onToggleClick = () => {
+    setIsExpanded(!isExpanded);
+  };
+
+  const panelContent = (
+    <DrawerPanelContent>
+      <DrawerHead>
+        <span tabIndex={isExpanded ? 0 : -1} ref={drawerRef}>
+          drawer-panel
+        </span>
+        <DrawerActions>
+          <DrawerCloseButton onClick={onCloseClick} />
+        </DrawerActions>
+      </DrawerHead>
+    </DrawerPanelContent>
+  );
+
+  return (
+    <DashboardWrapper breadcrumb={null} mainContainerId="scrollable-element">
+      <Drawer isExpanded={isExpanded}>
+        <DrawerContent panelContent={panelContent} id="jump-links-drawer-drawer-scrollable-container">
+          <DrawerContentBody>
+            <DrawerContentBody hasPadding>
+              <Sidebar>
+                <SidebarPanel variant="sticky">
```

Comment:
The `<SidebarPanel variant="sticky">` is jumping on scroll because `--pf-c-sidebar__panel--Top` is set to `0`. To correct this, either the `.pf-c-sidebar__panel` has to receive padding or `--pf-c-sidebar__panel--Top` has to be changed to match the `.pf-c-drawer__body.pf-m-padding` padding value.